interfaces/Portal/src/app/pages/dashboard/dashboard.page.html

```diff
@@ -1,4 +1,4 @@
-<app-header [title]="'page.dashboard.pageTitle'|translate"></app-header>
+<app-header [title]="'page.dashboard.pageTitle' | translate"> </app-header>
```

Comment:
Weird that these auto-format issues wheren't there before...

The Portal already had Prettier v3.2.5... so previous runs of the linting should have highlighted these issues... :?

Reply:
Its sort-of the opposite of what happend on the `index.html`... :?

Reply:
Probably a side-effect of the mismatch in config-for-eslint by the missing "eslint-plugin-config-prettier" (or the other one).